### DISCUSSION TITLE
cmdstar can not find csv in new session

### DISCUSSION THREAD

**Discussion by yonicd** [wlandau/stantargets#44] [Help]
I ran a stan model a while back using cmdstanr and stantargets and have now tried to use tar_load to start working again on model objects that are cached.

I am running into a problem that the csv's that are associated with the cmdstanr object are missing, or at least cmdstanr is looking for them in the wrong place (tempdir()). 

does stantargets store the csv's in another place when caching?

**Comment by wlandau** [comment]
`stantargets` tries to avoid the need for CSV files altogether. Before the CmdStanFit object is stored, `stantargets` reads in the contents into the object so those CSV files are no longer necessary. This is cleaner than making `targets` micromanage CSV files outside `_targets/`.

https://github.com/ropensci/stantargets/blob/ec5b5e283a836705bbb9817db892c6f8684fec4f/R/tar_stan_mcmc.R#L477-L483

**Comment by yonicd** [comment]
I wanted to use {loo} which needs a stanfit objects. This object can be created from the new R6, but it looks like the csv's are needed for it.

**Reply by yonicd** [reply]
I see now. `loo::loo(fit$draws())`
